python/smqtk/detect_and_describe/__init__.py

```diff
@@ -0,0 +1,64 @@
+import abc
+import os
+
+from smqtk.data_rep import DataElement
+from smqtk.utils.plugin import get_plugins
+
+class DetectAndDescribe (object):
+	def __init__(self, descriptor_element_factory, data_element_type):
```

Comment:
We're not doing anything with these parameters, so I don't think this super-constructor needs to exist.